### ISSUE TITLE
Consider adding support for CMake or package managers

### ISSUE THREAD

**Issue by rbrugo** [Iskustvo/i3-ipcpp#6]
Hi,
I think it would be really cool to have a way to easily integrate this library as a dependency in other projects, for example as a cmake submodule (with `find_package(i3-ipcpp)` ) or as conan package.

**Comment by Iskustvo** [comment]
Hi @rbrugo,
Unfortunately I am not that familiar (nor that interested, to be honest) with CMake and Conan.
When I wrote the library, the idea was for it to be packaged by the Linux distros if people find it useful.
I wanted to write AUR package myself but never actually did it :(
(In the meantime I started redesigning and refactoring this library locally and stopped working as I don't have too much free time at the moment, so everything is in my TODO list, but barely progresses over time)

That being said, I'll be glad to assist you if you are willing to do the heavy lifting with adding that support in this repo.
